### ISSUE TITLE
VMCloak - missing dependencies

### ISSUE THREAD

**Issue by Brueggus** [BlackArch/blackarch#2474]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended

----------------------------------------

### Bug description
After installing vmcloak from the blackarch repository, I get the following errors:

```
pkg_resources.DistributionNotFound: The 'alembic<1.1,>=1.0.7' distribution was not found and is required by VMCloak
pkg_resources.DistributionNotFound: The 'sqlalchemy==1.3.3' distribution was not found and is required by VMCloak
pkg_resources.DistributionNotFound: The 'pyyaml==3.12' distribution was not found and is required by VMCloak
```

### Steps to reproduce
- pacman -S vmcloak
- vmcloak init --verbose --win7x64 win7x64base --cpus 2 --ramsize 4096

**Actual result:** Describe here what happens after you run the steps above (i.e. the buggy behaviour)

I get the following errors:

```
pkg_resources.DistributionNotFound: The 'alembic<1.1,>=1.0.7' distribution was not found and is required by VMCloak
pkg_resources.DistributionNotFound: The 'sqlalchemy==1.3.3' distribution was not found and is required by VMCloak
pkg_resources.DistributionNotFound: The 'pyyaml==3.12' distribution was not found and is required by VMCloak
```

**Expected result:** Describe here what should happen after you run the steps above (i.e. what would be the correct behaviour)

I do not get the errors mentioned above.

### Info for developers

**Device:** Manufacturer Model XVI (Only if you consideer needed, for example, network cards)
**GNU/Linux distribution:** (If you are using one distro that is not BlackArch, example: Manjaro, Anarchy, etc)
**Tool version:** vmcloak (2:0.4.8-1)

**Comment by noraj** [comment]
- VMCloak PKGBUILD: https://github.com/BlackArch/blackarch/blob/master/packages/vmcloak/PKGBUILD
- VMcloak upstream requirements.txt: https://github.com/hatching/vmcloak/blob/master/requirements.txt

I should try it with blackarch-devtool and see.

**Comment by Brueggus** [comment]
Yes... the requirements.txt seems to be outdated. :|

https://github.com/hatching/vmcloak/blob/master/setup.py might help. They seem to be enforcing specific versions though.

**Comment by noraj** [comment]
I don't understand we are building and installing setup.py in a folder with `python2 setup.py install --root="$pkgdir" --skip-build`. Doing what's in the PKGBUILD manually in a virtualenv works. But it's not working in a BA sandbox:

```
$ blackarch-devtools -t -e 'vmcloak init --verbose --win7x64 win7x64base --cpus 2 --ramsize 4096' -p ./vmcloak-2:0.4.8-1-any.pkg.tar.xz 
loading packages...
resolving dependencies...
looking for conflicting packages...

Packages (1) vmcloak-2:0.4.8-1

Total Installed Size:  0,91 MiB

:: Proceed with installation? [Y/n] 
(1/1) checking keys in keyring                                                                                                                 [########################################################################################] 100%
(1/1) checking package integrity                                                                                                               [########################################################################################] 100%
(1/1) loading package files                                                                                                                    [########################################################################################] 100%
(1/1) checking for file conflicts                                                                                                              [########################################################################################] 100%
(1/1) checking available disk space                                                                                                            [########################################################################################] 100%
:: Processing package changes...
(1/1) installing vmcloak                                                                                                                       [########################################################################################] 100%
:: Running post-transaction hooks...
(1/1) Arming ConditionNeedsUpdate...
Package ./vmcloak-2:0.4.8-1-any.pkg.tar.xz installed correctly! Testing it now...
Traceback (most recent call last):
  File "/usr/bin/vmcloak", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/usr/lib/python2.7/site-packages/pkg_resources/__init__.py", line 3250, in <module>
    @_call_aside
  File "/usr/lib/python2.7/site-packages/pkg_resources/__init__.py", line 3234, in _call_aside
    f(*args, **kwargs)
  File "/usr/lib/python2.7/site-packages/pkg_resources/__init__.py", line 3263, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/usr/lib/python2.7/site-packages/pkg_resources/__init__.py", line 583, in _build_master
    ws.require(__requires__)
  File "/usr/lib/python2.7/site-packages/pkg_resources/__init__.py", line 900, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/usr/lib/python2.7/site-packages/pkg_resources/__init__.py", line 786, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'alembic<1.1,>=1.0.7' distribution was not found and is required by VMCloak
```

**Comment by anunna** [comment]
Duplicate and was fixed already in d4447a5 years ago.
